src/composables/useApi.ts

```diff
@@ -0,0 +1,75 @@
+// libraries
+import { AxiosError } from 'axios';
+import { Notify } from 'quasar';
+import { api } from '../boot/axios';
+import { i18n } from '../boot/i18n';
+
+// config
+import { rideToWorkByBikeConfig } from '../boot/global_vars';
+const { apiVersion: defaultApiVersion } = rideToWorkByBikeConfig;
+
+// types
+import type { Method } from 'axios';
+
+interface ApiResponse<T> {
+  data: T | null;
+}
+
+export const useApi = () => {
+  const apiFetch = async <T>({
+    endpoint,
+    payload,
+    translationKey,
+    method = 'get',
+    apiVersion = defaultApiVersion,
+  }: {
+    endpoint: string;
+    payload: unknown;
+    translationKey: string;
+    method: Method;
+    apiVersion?: string;
+  }): Promise<ApiResponse<T>> => {
+    try {
+      const response = await api<T>({
+        url: endpoint,
+        method: method,
+        data: payload,
+        headers: {
```

Comment:
We need `headers` param as `apiFetch` func param with default header argument `{  Accept: `application/json; version=${apiVersion}` }`

Reply:
Ok, updated, so that the func param is the full header object.